apps/notifications/src/panel/templates/body.jsx

```diff
@@ -52,16 +52,16 @@ export const NoteBody = createReactClass( {
 	},
 
 	UNSAFE_componentWillMount: function () {
-		var note = this.props.note,
-			hasReplyBlock;
+		const note = this.props.note;
+		let hasReplyBlock;
 
 		if ( note.meta && note.meta.ids.reply_comment ) {
 			hasReplyBlock =
 				note.body.filter( function ( block ) {
 					return (
 						block.ranges &&
 						block.ranges.length > 1 &&
-						block.ranges[ 1 ].id == note.meta.ids.reply_comment
+						block.ranges[ 1 ].id === note.meta.ids.reply_comment
```

Comment:
I couldn't find docs about the type of `reply_comment`, but https://github.com/Automattic/wp-calypso/blob/fix/lint-app-notifications/apps/notifications/src/panel/templates/functions.jsx#L215 is doing the same check and they use `===` there, so I guess it is safe.